client/src/diff/TextDiff.js

```diff
@@ -114,31 +116,36 @@ const process_indicators = (matched_indicators, indicator_status) => {
       };
     })
     // target_changed is subset of indicator_desc_changed so it must be included
-    .filter(row => indicator_status["indicator_desc_changed"].active && row.status === "target_changed" ?
-            true : indicator_status[row.status].active)
+    .filter(
+      row => indicator_status["indicator_desc_changed"].active && 
+        row.status === "target_changed" ?
+          true :
+          indicator_status[row.status].active
+    )
```

Comment:
This redesign makes the ternary seem separate from the `indicator_status["indicator_desc_changed"].active`, but the test for the ternary is the full `indicator_status["indicator_desc_changed"].active && row.status === "target_changed"`

Reply:
Does
```
.filter(
  row => indicator_status["indicator_desc_changed"].active && 
    row.status === "target_changed" ? true : indicator_status[row.status].active
)
```
read more clearly?

When I want to wrap an expression to multiple lines, I try to break after an operator add a level of indentation after the first wrap. I also prefer to always break ternaries in to three lines, but I see how that scans oddly here.

Reply:
Line seems to be part of a bigger bug, #280, #wontfix in this PR